LICENSE

```diff
@@ -2308,3 +2308,56 @@ distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
+For the QuickJS component couch_js/quickjs/quickjs:
```

Comment:
Why is there a second `/quickjs` in `couch_js/quickjs/quickjs`?

Reply:
Oh good find, Jay. That was the old location some time ago when quickjs was in the `couch_js/quickjs/quickjs` folder. I'll update it to be `src/couch_quickjs/quickjs`

Reply:
Updated the LICENSE file. Thanks for noticing, Jay!